2682. Find the Losers of the Circular Game/solution.py

```diff
@@ -0,0 +1,45 @@
+class Solution:
+    # n friends are playing a game where they pass a ball to each other. each round,
+    # they pass the ball roundNum*k indices to the right. this function calculates
+    # which friends lost the game. the winners get the ball, and the game ends when
+    # one friend gets the ball twice
+    def circularGameLosers(self, n: int, k: int) -> List[int]:
+        # define the list of friends with a range function. n+1 is due to range()
+        # being uninclusive in Python
+        friends = list(range(1, n + 1))
+        
+        # define the number of rounds, starting from 0
+        roundNum = 0
+        
+        # define who gets the first ball, starting at index 0
+        ballIndex = 0
+        
+        
+        # iterate until a friend gets the ball twice. the only way is to loop forever
+        # and then break the loop when a friend gets the ball
+        while True:
+            # calculate where the ball is going to go next
+            ballIndex = (ballIndex + roundNum * k) % n
```

Comment:
Order of operation can be ambiguous. Better to use extra parentheses.

Reply:
Oh you want me to put parens around "roundNum * k", right?